HW7/task34.py

```diff
@@ -0,0 +1,36 @@
+# Вам стоит написать программу. Винни-Пух считает, что ритм есть,
+#  если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое.
+#  Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+# Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры. 
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+#     **Вывод:** Парам пам-пам  
+
+
+def count_syllables(word):
+    cnt = 0
+    for i in word:
+        if i in ['а', 'и', 'е', 'о', 'у', 'э', 'ю', 'я', 'ы', 'ё']:
```

Comment:
Список не нужен, можно просто использовать строку.
```suggestion
        if i in 'аяуюоеёэиы':
```